### DISCUSSION TITLE
Connection of ZOWE to Mainframes isn't working.

### DISCUSSION THREAD

**Discussion by ssshreyas34** [zowe/zowe-explorer-vscode#3187] [Q&A]
Hello Team,

It might sound preposterous since it is a very basic doubt but recently I have started exploring ZOWE in VS code and was trying to setup the connection but somehow the connection is not getting establishing even though I have prompted all the required information like HOST, PORT. ACCOUNT NO and other relevant details. We are not facing any error but at the bottom right it displays Zosmf connecting but it is not. We have explored all the options and did all type of configuration we could think of but still no luck. Can you help us out?

Thanks, 
Happy Learning.

**Comment by adam-wolfe** [comment]
Hi @ssshreyas34, the first thing I would suggest is to look at the logs, which are stored in `[USER_HOME_FOLDER]\.vscode\extensions\zowe.vscode-extension-for-zowe-3.0.0\logs`, particularly `imperative.log`.

If the logs do not show any obvious problems. I would suggest trying to connect with `rejectUnauthorized` set to `false` in your config.
